pkg/kubelet/cm/pod_container_manager_linux.go

```diff
@@ -110,6 +110,10 @@ func (m *podContainerManagerImpl) EnsureExists(pod *v1.Pod) error {
 	return nil
 }
 
+func (m *podContainerManagerImpl) GetEventDispatcherChan() chan EventDispatcherEvent {
+	return m.eventDispatcher.GetEventChannel()
```

Comment:
if `cgrous-per-qos` flag is false (or future flag used to enable ext-iso) then there is no `m.eventDispatcher` object present